packages/nimble-components/src/combobox/tests/combobox.foundation.spec.ts

```diff
@@ -0,0 +1,638 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
Whatever we decide about policies for forked tests [in this thread](https://github.com/ni/nimble/pull/1256#discussion_r1198325456) should be applied to these ones too.

Reply:
Done